ocp_resources/virtual_machine_instance.py

```diff
@@ -124,6 +125,9 @@ def wait_until_running(self, timeout=TIMEOUT_4MINUTES, logs=True, stop_status=No
 
```

Comment:
if vmi has a vm please also log vm status

Reply:
vmi doesn't have vm attribute. will need to add new property for it

Reply:
VirtualMachine.get(name=self.name, namespace=self.namespace) ....

Reply:
done

Reply:
the comment was for the exception part :)  leave the above log and add also right after `except TimeoutExpiredError`